src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -411,12 +414,38 @@
           Returns="@(PackageFile)"
           DependsOnTargets="GetFiles;EnsureOOBFramework;$(VersionDependsOn)">
     <ItemGroup>
-      <PackageFile Include="@(File)" />
+      <!-- Include all files except source files. Sources need to be deduplicated. -->
+      <PackageFile Include="@(File)" Condition="'%(File.IsSourceCodeFile)'!='true'" />
       <PackageFile Condition="'%(PackageFile.PackageId)' == ''">
         <PackageId>$(Id)</PackageId>
         <PackageVersion>$(Version)</PackageVersion>
       </PackageFile>
 
+      <!-- Include Symbols -->
+      <PackageFileSymbol Condition="$([System.String]::new('%(PackageFile.Identity)').EndsWith('.dll'))"
```

Comment:
I see, yeah, this was part of @chcosta's prototype of making the symbol packages and I didn't think to change it after figuring out `PackageLibs.targets`.

On that note I forgot to cc you on the review @chcosta, sorry!